src/main/java/rs117/hd/data/WaterType.java

```diff
@@ -153,7 +153,7 @@ public enum WaterType
 		.fresnelAmount(.2f)
 		.surfaceColor(rgb("#07292f"))
 		.foamColor(rgb(64, 64, 64))
-		.depthColor(rgb("#000000"))
+		.depthColor(rgb("#ffffff"))
```

Comment:
Was this change intended?
Before:
<img width="1152" height="777" alt="image" src="https://github.com/user-attachments/assets/e32a530e-2479-4000-ad9c-de6492706003" />
After:
<img width="1152" height="777" alt="image" src="https://github.com/user-attachments/assets/40445f3b-5170-4ac0-b4d4-592599e283d0" />